setup.cfg

```diff
@@ -1,72 +1,8 @@
-[metadata]
-name = goodconf
-version = 2.0.2.dev0
-description = Load configuration variables from a file or environment
-long_description = file: README.rst
-author = Peter Baumgartner
-author_email = pete@lincolnloop.com
-url = https://github.com/lincolnloop/goodconf/
-keywords = env, config, json, yaml
-license = MIT
-classifiers =
-    Development Status :: 5 - Production/Stable
-    Intended Audience :: Developers
-    License :: OSI Approved :: MIT License
-    Operating System :: OS Independent
-    Programming Language :: Python
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
-    Programming Language :: Python :: 3.10
-    Programming Language :: Python :: 3.11
-
-[options]
-packages = find:
-zip_safe = false
-install_requires =
-    pydantic>=1.0
-
-[options.package_data]
-goodconf = py.typed
-
-[options.packages.find]
-exclude = tests
-
-[options.extras_require]
-yaml =
-    ruamel.yaml>=0.17.0
-toml =
-    tomlkit>=0.11.6
-
-tests =
-    django>=3.2.0
-    ruamel.yaml>=0.17.0
-    tomlkit>=0.11.6
-    pytest==7.1.*
-    pytest-cov==2.5.1
-    pytest-mock==1.7.1
-maintainer =
-    zest.releaser[recommended]
-
-[bdist_wheel]
-universal=1
-
-[tool:pytest]
-addopts = --cov
-
 [zest.releaser]
 tag-signing = yes
 date-format = %%-d %%B %%Y
 tag-format = v{version}
 
-[check-manifest]
-ignore =
-    tests
-    tests/*
-
-[coverage:run]
-source = goodconf
-
-[coverage:report]
-show_missing = true
-skip_covered = true
+[flake8]
```

Comment:
Flake8 doesn't support pyproject.toml yet, I could only move it into `.flake8`, but as long as zest stays we can keep setup.cfg?

Reply:
Yeah, we can keep `setup.cfg` for now. Moving to `ruff` (via [`flake8-to-ruff`](https://pypi.org/project/flake8-to-ruff/) might be an option, but that's low priority.

dropping this here in case the situation changes: https://github.com/PyCQA/flake8/issues/234